### PR TITLE
[website] Update showcase

### PR DESCRIPTION
**Pull request by cornhundred**
Adding [Vizgen single cell spatial viewer](https://github.com/visgl/deck.gl/discussions/6061) to the deck.gl showcase.

### CODE REVIEW

File: website/showcase.json [hunk 1]
```diff
@@ -1,4 +1,13 @@
 [
+  {
+    "name": "Vizgen Single-Cell MERFISH Spatial Visualizer",
+    "url": "https://vizgen.com/applications/#visual",
```
Comment (ibgreen) [inline website/showcase.json:4]: This link seems to goes to a pretty generic/busy page - hard to tell what the demo is here. https://vizgen.com/applications/#visual

Comment (cornhundred) [reply website/showcase.json:4]: Hi @ibgreen, yeah we're putting in an anchor tag which should be up by tomorrow. This new tag will direct to the visual which is in the middle of the page.

Comment (cornhundred) [reply website/showcase.json:4]: Hey @ibgreen, the anchor tag should be functional now and direct to the visual at the link https://vizgen.com/applications/#visual 

Let us know if you would like us to modify anything.

File: website/showcase.json [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+    "image": "/images/showcase/vizgen_single_cell_spatial.jpg",
+    "links": {
+      "by Vizgen": "https://vizgen.com/"
+    },
+    "description": "Vizgen's Interactive Single-Cell Spatial Visualizer enables users to explore single-cell gene expression data from our MERFISH Mouse Brain Receptor Map. Cells can be visualized using a UMAP embedding or the spatial coordinates within the brain - providing a detailed spatial map of the cell clusters and gene expression patterns in the tissue. Users can select a gene of interest from the dropdown list, click individual cells to highlight the cell clusters, and toggle between UMAP and spatial views."
```
Comment (Pessimistress) [inline website/showcase.json:9]: This text is too long and will not all fit in the template.

Comment (cornhundred) [reply website/showcase.json]: Hi @Pessimistress we shortened the description, let us know if you need anything else.

**Review by Pessimistress — APPROVED** [review]

**Pull request merged**